APIBacklog/documentation/SupportingDocuments/API proposals/APIproposal_Traffic Influence_TIM.md

```diff
@@ -0,0 +1,13 @@
+| **Field** | Description | 
+| ---- | ----- |
+| API name | Traffic Influence |
+| API owner | TIM, TEF |
```

Comment:
API owner is the proponent company, which is this case is TIM. So I suggest removing TEF.

Reply:
API Description modified removing TEF as owner